committee-code-of-conduct/charter.md

```diff
@@ -0,0 +1,148 @@
+however, will retroactively review any action taken in such instances to ensure
+it was appropriate.
+
+The committee meets biweekly unless additional interstitial meetings are
+required to address incidents or other critical work. Meetings are not recorded,
+however confidential notes may be kept when necessary to provide continuity to
+future committee members. Wherever possible, documentation necessary for the
+internal operation of the committee will be stored in a private GitHub
+repository.
+
+### Meeting quorum
+Meetings are considered at quorum when a simple majority of the members are
+present. Where there are 4 or fewer members available due vacant seats or
+recusal, quorum is 2.
+
+### Policy change ratification
+Any changes to the charter require explicit LGTM or Approve from all
+committee members. For pull requests, a /hold will be applied until all
+approvals are present. Any changes merged without consensus will be reverted.
+
+### Incident report confidentiality
+The Code of Conduct committee will keep your report confidential. The CoCC may
+share report information with the community chairs if they believe doing so is
+appropriate. Past incidents are communicated generally to new committee members
+so they can have historical context for future issues. While this may allow the
+establishment of bias in new members it is believed that the educational value
+of this information outweighs the potential negatives.
+
+### Incident Response Recusal
+A member of the Code of Conduct committee shall recuse themselves from
+evaluating and responding to any incident for which they are unable to be
+impartial. If a committee member does not recuse themselves they may be removed
+from participation by a unanimous vote of all other members of the committee.
+
+### Committee seats unoccupied
+In the event that one or more of the seats on the committee is unoccupied, for any
+reason, a replacement member will be appointed by the community chairs as soon
+as reasonable. That person will serve out the remainder of the term of the
+person they are replacing.
+
+### Committee dissolution
+If committee members believe that the committee is no longer able to act in
+accordance with the above Mission/Purpose the committee may vote to dissolve.
+The committee should specify a date of dissolution. Dissolution requires an
+affirmative vote of more than 75% of committee members, and must be unamimous
+when the committee has only 4 or fewer members. If the committee is dissolved,
+all seats are vacated on the date specified.
+
+### Removal
+A committee member may be removed from the committee by a unanimous decision of
+the other committee members. The member should be given the opportunity to
+resign before they are removed. Removal should only be considered for the
+following reasons:
+
+* The member has been found to have committed a code of conduct violation.
+* The member is convicted of a felony.
```

Comment:
it is unclear why this is here... someone convicted of a felony that has server their time but otherwise abides by the CoC isn't welcome here?  

1. I think American's can be really hard on felons for reasons I don't quite understand. 
2. This is a document to serve the world correct?  by "felony" do we mean US law?  or are there equivalent labels for other sovereignties?

Reply:
Personally I think this condition should be removed.

Reply:
i think it should be removed as well.